### ISSUE TITLE
Checksum for blockchain download

### ISSUE THREAD

**Issue by stevenjack** [BTCPrivate/bitcoin-private-full-node-wallet#15]
Would like to verify the bulk download of the blockchain, are you able to update the current release with the checksum of the download link:

https://storage.googleapis.com/btcp-blockchain/BTCprivate.7z

**Comment by ch4ot1c** [comment]
Yes - I will add this to the BTCPrivate/BitcoinPrivate readme docs as well.

`curl https://s3.amazonaws.com/btcp.snapshot/utxo_snapshot.tar.gz | tar xvz`
